Documentation/operations/upgrade.rst

```diff
@@ -581,7 +601,7 @@ Upgrading from >=1.7.0 to 1.8.y
     .. parsed-literal::
 
       helm upgrade cilium --namespace=kube-system \\
-      --set global.bpf.ctTcpMax=100000
+      --set bpf.ctTcpMax=100000
```

Comment:
this should not be modified since this section referes to upgrade from 1.7 to 1.8, we aren't changing the helm chart of 1.8 branch.

Reply:
not addressed